monai/engines/utils.py

```
@@ -360,3 +361,119 @@ def default_metric_cmp_fn(current_metric: float, prev_best: float) -> bool:
+
+    Args:
+        accumulation_steps: number of mini-batches to accumulate before updating
+            weights. Must be a positive integer. Default: 2.
+
+    Raises:
+        ValueError: when ``accumulation_steps`` is not a positive integer.
+    """
+
+    def __init__(self, accumulation_steps: int = 2) -> None:
+        if not isinstance(accumulation_steps, int) or accumulation_steps < 1:
+            raise ValueError(f"`accumulation_steps` must be a positive integer, got {accumulation_steps!r}.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)